lib/packages.ts

```diff
@@ -164,7 +164,7 @@ function _getVersionFromGit(experimental: boolean): string {
   }
 
   const hasLocalChanges = _exec(`git status --porcelain`) != '';
-  const scmVersionTagRaw = _exec(`git describe --match v[0-9].[0-9].[0-9]* --abbrev=7 --tags`)
+  const scmVersionTagRaw = _exec(`git describe --match v[0-9]*.[0-9]*.[0-9]* --abbrev=7 --tags`)
```

Comment:
I think technically it should be `[0-9]+`, i.e. "one or more digits", but it's not really a big deal.

Reply:
`git describe`s `--match` flag is a glob rather than a regex, so a `+` is actually not what we are looking for.  We ran into this on framework as well.

Reply:
Yes, `+` does not work in glob and `extglobs` doesn't seem like an option.